src/pybind/mgr/dashboard/controllers/crush_rule.py

```diff
@@ -37,16 +37,23 @@ def get(self, name):
             if r['rule_name'] == name:
                 return r
         raise NotFound('No such crush rule')
-
-    def create(self, name, root, failure_domain, device_class=None):
-        rule = {
-            'name': name,
-            'root': root,
-            'type': failure_domain,
-            'class': device_class
-        }
-        CephService.send_command('mon', 'osd crush rule create-replicated', **rule)
-
+    def create(self, name, failure_domain, device_class, root=None, profile=None, pool_type=None):
```

Comment:
maybe we can default it to `replication` by default?
```suggestion
    def create(self, name, failure_domain, device_class, root=None, profile=None, pool_type='replication'):
```
```suggestion
    def create(self, name, failure_domain, device_class, root=None, profile=None, pool_type=None):
```